### PR TITLE
Temp series workflow

### PR DESCRIPTION
**Pull request by pashworth-tna**
New file for Temporary series workflow and creation of the BagIT export package

### CODE REVIEW

File: architecture-decision-records/0026-temporary_series_workflow.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+TDR application will support the creation of a temporary series by the user.
+Until a full series is defined for a consignment it cannot progress through to the preservation systems.
+
+## Decision
+
+To help support the preservation process it was decided that TDR would not export the BagIT package until a permanent Series was added to the TDR database.
+At which point TDR would export the BagIT and notify downstream systems in the usual manner.
+
+The advantages of doing this are:
+* The user can still upload and deliver through TDR without the need
```
Comment (TomJKing) [inline architecture-decision-records/0026-temporary_series_workflow.md:16]: Think there is some missing wording at the end of this point: "... TDR without the need _to wait for the assignment of a permanent series reference_"

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. Thanks

File: architecture-decision-records/0026-temporary_series_workflow.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# 26. Court Judgment Transfers Url Scheme
```
Comment (TomJKing) [inline architecture-decision-records/0026-temporary_series_workflow.md:1]: Not sure this title is correct?

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

File: architecture-decision-records/0026-temporary_series_workflow.md (hunk 1 above)
Comment (pashworth-tna) [reply architecture-decision-records/0026-temporary_series_workflow.md]: Sorry. Hopefully sorted now

**Pull request merged**